en/components/general/ignite-ui-licensing.md

```diff
@@ -63,17 +62,35 @@ This will allow you to seamlessly use a mix of packages from the public npm regi
 >[!NOTE]
 > `npm` is disallowing the use of the `"@"` symbol inside your username as it is considered as being "not safe for the net". Because your username is actually the email that you use for your Infragistics account it always contains the symbol `"@"`. That's why you must escape this limitation by replacing the `"@"` symbol with `"!!"` (two exclamation marks). For example, if your username is `"username@example.com"` when asked about your username you should provide the following input: `"username!!example.com"`.
 
-#### Now, to log in to our private feed using npm, run the adduser command and specify a user account and password:
+### Now, to log in to our private feed using npm
 
-```cmd
-npm adduser --registry=https://packages.infragistics.com/npm/js-licensed/ --scope=@infragistics --always-auth
-```
->[!NOTE]
-> If you are using `npm` version 9.0.0 or higher you need to set `--auth-type=legacy`
+#### Node version up to v8
+Run the `adduser` command and specify a user account and password:
 
-```cmd
+<pre style="background:#141414;color:white;display:inline-block;padding:16x;margin-top:10px;font-family:'Consolas';border-radius:5px;width:100%">
+npm adduser --registry=https://packages.infragistics.com/npm/js-licensed/ --scope=@infragistics
+</pre>
+
+#### Node v9.0.x - 9.3.x
+
+If you are using `npm` version 9.0.x till version 9.3.x you need to set `--auth-type=legacy`
+
+<pre style="background:#141414;color:white;display:inline-block;padding:16x;margin-top:10px;font-family:'Consolas';border-radius:5px;width:100%">
 npm adduser --registry=https://packages.infragistics.com/npm/js-licensed/ --scope=@infragistics --always-auth --auth-type=legacy
-```
+</pre>
+
+#### Node version 9.4+
```

Comment:
Btw, you can mention that this will also work for all the previous versions as well. Order the info in descending order. 9.4+ down.